server/datastore/mysql/microsoft_mdm.go

```diff
@@ -1874,3 +1899,26 @@ host_uuid = ? AND profile_name NOT IN(?) AND NOT (operation_type = '%s' AND COAL
 	}
 	return profiles, nil
 }
+
+func (ds *Datastore) WipeHostViaWindowsMDM(ctx context.Context, host *fleet.Host, cmd *fleet.MDMWindowsCommand) error {
```

Comment:
@ghernandez345 This is one new Datastore method you'll need to mock in the CLI PR for the Wipe test (windows hosts).